homeassistant/components/sensor/zamg.py

```diff
@@ -0,0 +1,251 @@
+# coding=utf-8
```

Comment:
Remove the whole utf and created by header please. The doc string should be the first part.